src/components/Calculator.js

```diff
@@ -1,41 +1,139 @@
 import React, { Component } from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: 0,
+      next: null,
+      operation: null,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    if (!e.target.name) return;
+    const { next, total, operation } = calculate(this.state, e.target.name);
+    this.setState({ next, total, operation });
+  }
+
   render() {
+    /* eslint-disable */
```

Comment:
- [ ] It is advisable that when we disable eslint it should only be a single rule and not eslint itself. We are also expected to state why we did that in a note to the reviewer. Please re-enable eslint.